secant.xcodeproj/xcshareddata/xcschemes/secant-mainnet.xcscheme

```diff
@@ -31,7 +31,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Debug"
+      buildConfiguration = "Release"
```

Comment:
I don't know if this is an intentional change, or a side-effect of reconfiguring something in XCode for testing this PR.

Reply:
Good catch, not ideal to leave release. Reverted.